src/server/http/index.js

```diff
@@ -38,7 +38,21 @@ export default async function (kbnServer, server, config) {
   server.connection({
     host: config.get('server.host'),
     port: config.get('server.port'),
-    listener: kbnServer.newPlatform.proxyListener
+    listener: kbnServer.newPlatform.proxyListener,
+    state: {
+      strictHeader: false,
+    },
+    routes: {
```

Comment:
__note:__ the browser tests revealed the problem I haven't noticed before - if cors is only configured in new platform then all requests that are forwarded to the legacy platform assume that cors is disabled that breaks browser tests. Soooo, it seems we still should preserve some part of server configuration for the legacy platform.

Reply:
Yeah, we can probably solve this duplication in a more sophisticated way, but I feel like proper comments in both places should work just fine. I'll add these comments, but let me know if you prefer something more than that.